spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe QueueItem do 
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+  
+  describe '#video_title' do 
```

Comment:
There is one nifty way how to even refactor this spec - check out shoulda matchers documentation and `delegate_method` matcher :). You should be able to build 1 liner expectation from this.